src/base/bittorrent/session.h

```diff
@@ -61,9 +61,16 @@ class ResumeDataSavingManager;
 
 enum MaxRatioAction
 {
-    Pause,
-    Remove,
-    EnableSuperSeeding
+    Pause = 0,
+    Remove = 1,
+    DeleteFiles = 3,
+    EnableSuperSeeding = 2
```

Comment:
Maybe I'm completely wrong, but it should be like this =
```
    Pause = 0,
    Remove = 1,
    DeleteFiles = 2,
    EnableSuperSeeding = 3
```
Since it like that here =
```
    const QHash<MaxRatioAction, int> actIndex = {
        {Pause, 0},
        {Remove, 1},
        {DeleteFiles, 2},
        {EnableSuperSeeding, 3}
    };
```

Reply:
>Maybe I'm completely wrong

Yes.